From 6fce3b3afda68cd041cb3b38c8c21fa0513ab5e3 Mon Sep 17 00:00:00 2001
From: Michael Simons <msimons@microsoft.com>
Date: Mon, 2 Aug 2021 19:56:17 +0000
Subject: [PATCH] Exclude warnings as errors from source build


NU1604:  These hopefully go away once runtime-portable is added to the tarball.
    Microsoft.AspNetCore.App.Runtime.csproj : error NU1604: Project dependency Microsoft.NETCore.App.Crossgen2.linux-x64 does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results.
    Microsoft.AspNetCore.App.Runtime.csproj : error NU1604: Project dependency Microsoft.NETCore.App.Runtime.linux-x64 does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results.

CS8601:
   CascadingAuthenticationState.razor(4,108): error CS8601: Possible null reference assignment.
---
 Directory.Build.props | 4 ++++++++++++++
 1 file changed, 4 insertions(+)

diff --git a/Directory.Build.props b/Directory.Build.props
index ba1c1675a0..ce6bc50c66 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -225,6 +225,10 @@
     <ArchiveExtension Condition="'$(TargetOsName)' == 'win'">.zip</ArchiveExtension>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);NU1604;CS8601</NoWarn>
+  </PropertyGroup>
+
   <Import Project="eng\Workarounds.props" />
   <Import Project="artifacts\bin\GenerateFiles\Directory.Build.props" Condition=" '$(MSBuildProjectName)' != 'GenerateFiles' " />
   <Import Project="eng\Dependencies.props" />
-- 
2.29.2

